terraso_backend/apps/core/models/landscapes.py

```diff
@@ -40,6 +43,26 @@ class Landscape(SlugModel):
     )
     groups = models.ManyToManyField(Group, through="LandscapeGroup")
 
+    area_types = models.JSONField(blank=True, null=True)
+    taxonomy_terms = models.ManyToManyField(TaxonomyTerm)
+    population = models.IntegerField(blank=True, null=True)
+
+    PARTNERSHIP_STATUS_NO = "no"
+    PARTNERSHIP_STATUS_IN_PROGRESS = "in-progress"
+    PARTNERSHIP_STATUS_YES = "yes"
+    DEFAULT_PARTNERSHIP_STATUS = PARTNERSHIP_STATUS_NO
+
+    MEMBERSHIP_TYPES = (
+        (PARTNERSHIP_STATUS_NO, _("No")),
+        (PARTNERSHIP_STATUS_IN_PROGRESS, _("In Progress")),
+        (PARTNERSHIP_STATUS_YES, _("Yes")),
```

Comment:
question: do we need to use `_()` here? Are we doing any localization in the backend?

Reply:
@paulschreiber This is how the other fields with choices are handled, I think this is just used for the admin panel right now